### DISCUSSION TITLE
O Zed virou open source, e pode se tornar mais um editor queridinho dos devs (br-dev)

### DISCUSSION THREAD

**Discussion by giscus** [br-dev-org/site#31] [Post Discussions]
# O Zed virou open source, e pode se tornar mais um editor queridinho dos devs (br-dev)

O Zed, novo editor de código feito em Rust, tem ganhado destaque entre devs, e agora é open source. Também está ganhando suporte a Linux.

https://br-dev.org/publicacoes/o-zed-virou-open-source-e-pode-se-tornar-mais-um-editor-queridinho-dos-devs

**Comment by MardoqueuLS** [comment]
Achei sensacional o desempenho dele quando eu vi em um canal gringo que divulgou. Espero que tenha o mesmo tipo de desempenho em Linux! Parece muito promissor.
